fvm/src/state_tree.rs

```diff
@@ -579,8 +579,8 @@ impl Arbitrary for ActorState {
             code: cid,
```

Comment:
The `code` and `state` cid shouldn't be identical. And we don't want to duplicate the code for `Cid::arbitrary(g)`. If necessary, this PR should depend on the other PR that upgrades the `cid` dependency.

Reply:
ok https://github.com/filecoin-project/ref-fvm/pull/1107